### ISSUE TITLE
Account page makes a request with an invalid query string

### ISSUE THREAD

**Issue by hdodov** [payloadcms/payload#1472]
# Bug Report

I'm running Payload on AWS Lambda with Docker, connected to MongoDB Atlas and Amazon S3. Everything works, except the account page. The API URL is `/api/users/undefined`.

Upon inspecting DevTools, I can see that there's one failing request, resulting in HTTP 400. It appears that this happens due to an invalid request URL:

```
…/api/users/62fb90287bed9b334e530765?depth=0?locale=en&fallback-locale=null
```

![image](https://user-images.githubusercontent.com/5570098/203098692-72452e97-b417-4423-98e6-d4d5adc9d68a.png)

As you can see, there are two question marks, meaning that only `?locale=en&fallback-locale=null` is interpreted as the query string, while `?depth=0` remains a part of the path. This means that `62fb90287bed9b334e530765?depth=0` is requested as the ID, which is obviously invalid.

## Other Details

Payload version: **1.2.0**

**Comment by jmikrut** [comment]
Hey @hdodov — very interesting. We'll get someone on this ASAP. Thanks for the report!

**Comment by hdodov** [comment]
Interestingly, the request has two question marks locally too, but it returns HTTP 200:

![image](https://user-images.githubusercontent.com/5570098/203105096-fb9facf8-1ecd-44ec-817f-09b3b368c6ed.png)

Perhaps that happens because the production traffic is proxied by AWS Lambda, which normalizes the request URL somehow.

Here's where the request was initiated:

![image](https://user-images.githubusercontent.com/5570098/203104855-eba3e837-3b93-4327-a52b-a58bd3d367c1.png)

**Comment by hdodov** [comment]
This happens even on [the live Payload demo site](https://demo.payloadcms.com/admin/account):

![image](https://user-images.githubusercontent.com/5570098/203106257-c6d1cb04-5e70-4b49-a9cb-22d1ff9ba35e.png)

This means it's not a bug in my environment setup. It's just that other environments handle the invalid URL differently and yield the expected result. But the URL still remains invalid.

**Comment by DanRibbens** [comment]
Released in v1.2.1

Thanks for reporting!

**Comment by hdodov** [comment]
Thank you very much for the quick response! You rock!
